src/printer.c

```diff
@@ -84,7 +84,15 @@ ly_print(struct lyout *out, const char *format, ...)
 
     switch(out->type) {
     case LYOUT_FD:
+#ifdef HAVE_VDPRINTF
         count = vdprintf(out->method.fd, format, ap);
+#else
+	{
+            FILE *stream = fdopen(out->method.fd, "r");
+            count = vfprintf(stream, format, ap);
+            fclose(stream);
+        }
+#endif
```

Comment:
Why the curly brackets? We try to keep case code without redundant indents so please remove them. Also, `fdopen` error checking would be nice.

Reply:
I created a new block because I needed to declare a new variable (`FILE *stream`). But I agree it's ugly, I'll add another `#ifdef` and declare the variable in the top of the function.

Reply:
Right, forgot about the variable declaration but it will probably be better to put it separately anyway.